Doc/library/argparse.rst

```diff
@@ -636,6 +636,25 @@ be positional::
    usage: PROG [-h] [-f FOO] bar
    PROG: error: the following arguments are required: bar
 
+By default, argparse automatically handles the internal naming and
+display names of arguments, simplifying the process without requiring
```

Comment:
@serhiy-storchaka , `of arguments` or `of non-positioal arguments`?
https://stackoverflow.com/a/20250435/556958